### PR TITLE
Added b2bTrialAnon Component

### PR DESCRIPTION
**Pull request by Yoav-Shafir**
🐿 v2.12.3

### CODE REVIEW

**Comment by liamkeaton** [comment]
This is failing on linting, make sure you've setup .editorconfig on your IDE so it sets tabs and spaces correctly for these files.

**Comment by CourtneyLO** [comment]
Have you checked that the banner shows when using your changes in next-messaging-guru?

File: manifest.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
-	licenceJoinPromptSsiClient: { // name of flag variant and must match the messageId in ammit-api
-		path: 'bottom/lazy', // path to template
```
Comment (CourtneyLO) [inline manifest.js]: Why did you change this?
